testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -41,11 +41,18 @@
     SUPPORTS_CREATE_TABLE,
     SUPPORTS_CREATE_TABLE_WITH_DATA(SUPPORTS_CREATE_TABLE),
 
+    SUPPORTS_COMMENT_ON_TABLE,
```

Comment:
I'm wondering. Can this actually be moved to the connector level?

Reply:
I don't fully understand the suggestion. Can you please exemplify?

Reply:
Can the connector itself expose the information which behaviours/functions/pushdowns it has implemented?

Reply:
In a way -- the connector does it, in its tests.

I do not see the benefit of moving this into production, plugin, code. Note that the behaviors defined here are modelled after testability and tests' need.
If one day we realize those do not feed the needs, we can define different options, or get rid of the enum at all.